### DISCUSSION TITLE
How to custom CSS class in table package ?

### DISCUSSION THREAD

**Discussion by t-prod** [filamentphp/filament#12447] [Help]
### Package

Table builder

### Package Version

v3.2.0

### How can we help you?

Hi,

I'm using the Filament table package in a standalone app with a custom Tailwind theme.
Is there a simplier way than override each files in vendor\filament\tables\resources\views\components ?
I see we can override attributes in each on components. Maybe we can do it the class when we define the table itself ?

**Comment by leandrocfe** [comment]
https://filamentphp.com/docs/3.x/support/style-customization#overview

You may be tempted to publish the internal Blade views to your application so that you can customize them. We don't recommend this, as it will introduce breaking changes into your application in future updates. Please use the [CSS hook classes](https://filamentphp.com/docs/3.x/support/style-customization#applying-styles-to-hook-classes) wherever possible.

**Comment by t-prod** [comment]
Thanks, indeed customing by CSS is really simpler I did not see this section in documentation ;)
